apps/server/src/modules/common-cartridge/controller/dto/course-export-body.response.ts

```diff
@@ -1,12 +1,12 @@
-import { CourseCommonCartridgeMetadataDto } from '../../common-cartridge-client/course-client';
+import { CourseCommonCartridgeMetadataResponse } from '@infra/courses-client';
 import { CourseFileIdsResponse } from './common-cartridge.response';
 
 export class CourseExportBodyResponse {
-	courseFileIds?: CourseFileIdsResponse;
+	public courseFileIds?: CourseFileIdsResponse;
 
-	courseCommonCartridgeMetadata: CourseCommonCartridgeMetadataDto;
+	public courseCommonCartridgeMetadata: CourseCommonCartridgeMetadataResponse;
```

Comment:
If they are not static, why did you change them to public? If they are to be directly read, may I suggest declaring it `readonly`

Reply:
In JavaScript/Typescript, the default is 'public'. I just made it explicit.